deploy/router/simple-setup

```diff
@@ -16,3 +16,6 @@ for p in cloud1 cloud2 edge1 edge2 edge3; do
     oc apply -f $ABSBASE/$p.yaml;
 done
 
+# route create by operator doesn't work over plain http yet and tls has not yet been setup
+oc expose -n edge1 service edge1 --name http --port 8672
+oc expose -n cloud1 service cloud1 --name console --port 8672
```

Comment:
(I took this out because on openshiftthis is no longer required, but forgot that on kubernetes ingress setup is not yet in place. Does oc expose work on kubernetes though?)

Reply:
Let me retest this as well.

Reply:
Works fine on minishift. Let's keep it simple

Reply:
So if you are on openshift then the operator should already be creating routes for the console in each namespace. Is this not happening for you? If not what is the image has you are using for the operator? (and are there any errors in the logs from the operator pod in one of the namespaces?)

Reply:
Yeah, it's all working fine. I removed it from PR